### PR TITLE
Get logs from tests against docker

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

# Overview

- [ ] Get logs from tests against docker

# Comments

There were some failures on latest docker run:
https://github.com/input-output-hk/cardano-wallet/runs/2108457759?check_suite_focus=true

Unfortunately there are no logs uploaded from a docker runs currently (like they are from Win/Linux/Mac runs), so potential failures cannot be properly investigated. This change is to attach logs from docker also.

### CODE REVIEW

**Comment by piotr-iohk** [comment]
Logs should be now attached for docker workflow as well -> https://github.com/input-output-hk/cardano-wallet/actions/runs/654440971

File: test/e2e/.rake_tasks~ [hunk 1]
```diff
@@ -3,6 +3,7 @@ clean_logs
 fixture_wallets_decode
 fixture_wallets_encode
 fixture_wallets_template
+get_docker_logs
```
Comment (rvl) [inline test/e2e/.rake_tasks~:6]: What is this file?
I am used to the convention where files ending with `~` are backup files.
Is it hand-crafted, or auto-generated from `Rakefile`?
If the latter, then it should be removed and added to .gitignore.

File: test/e2e/Rakefile [hunk 2]
```diff
@@ -250,6 +250,27 @@ task :get_latest_configs, [:env] do |task, args|
...
+  puts "\n  >> Getting docker logs into #{LOGS}"
+
+  mk_dir(LOGS)
+  node_logfile = `docker inspect --format='{{.LogPath}}' cardano-node`.gsub("\n",'')
+  wallet_logfile = `docker inspect --format='{{.LogPath}}' cardano-wallet`.gsub("\n",'')
+
+  node_cmd = "sudo cp #{node_logfile} #{LOGS}/node.log"
+  wallet_cmd = "sudo cp #{wallet_logfile} #{LOGS}/wallet.log"
+
+  puts node_cmd
```
Comment (rvl) [inline test/e2e/Rakefile]: Is there a way for rake to automatically print commands before they are run?

**Review by rvl — APPROVED** [review]
Looks helpful, thanks.

File: test/e2e/.rake_tasks~ (hunk 1 above)
Comment (piotr-iohk) [reply test/e2e/.rake_tasks~:6]: It was autogenerated first but I've been adding new tasks to it. As far as I see rake uses it for autocompletion so it seemed useful to me to have it checked in, although it looks a bit awkward. There is another one .rakeTasks which seems to be autogenerated by Atom, and it's not useful. I'll remove that one.

Comment (piotr-iohk) [reply test/e2e/.rake_tasks~:6]: It was autogenerated first but I've been adding new tasks to it. As far as I see rake uses it for autocompletion so it seemed useful to me to have it checked in, although it looks a bit awkward. There is another one .rakeTasks which seems to be autogenerated by Atom, and it's not useful. I'll remove that one.

File: test/e2e/Rakefile (hunk 2 above)
Comment (piotr-iohk) [reply test/e2e/Rakefile]: I don't think so. I think I could wrap it in a `cmd` command or something like that.

Comment (piotr-iohk) [reply test/e2e/Rakefile]: I have added `cmd` method and wrapped all commands into it. (I've found there is a `sh` command that comes with rake, that does the same pretty much, however It was giving me strange failures on Windows for some reason)

**Pull request merged**